client/src/styles/leafletComponents/resultItem.less

```diff
@@ -63,4 +63,9 @@
 
     .c-result-item__sharing {
       padding-top: 7px;
-    }
\ No newline at end of file
+    }
+
+.fa {
+  padding: 15px;
+  padding-top: 22px;
+}
```

Comment:
ok, will do
On 06/17/2015 06:48 AM, lifehackett wrote:
>
> In client/src/styles/leafletComponents/resultItem.less 
> <https://github.com/MobilityLabs/medellin-heritage/pull/34#discussion_r32618168>:
>
> > \ No newline at end of file
> > +    }
> > +
> > +.fa {
> > +  padding: 15px;
> > +  padding-top: 22px;
> > +}
>
> You don't want to do this. This will impact every font-awesome icon on 
> the page, such as the magnifying glass on the search. Be extremely 
> cautious when making changes that have a big scope. Typically it is 
> better to decorate a specific item.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MobilityLabs/medellin-heritage/pull/34/files#r32618168>.
>